react/lib/altpayment/sideshift.ts

```diff
@@ -0,0 +1,82 @@
+import { AltpaymentClient } from ".";
+
+export const BASE_SIDESHIFT_URL = 'https://sideshift.ai/api/v2/'
```

Comment:
this could be in config file

Reply:
Not a matter for config IMO, this is not something we would be changing, not a URL that depends on the node/mirror/server, just the fixed sideshift API url. If they change it eventually, then we just change it here and commit it. Config should be for more volatile/optional stuff.

As for the naming: indeed, addressed.